node_modules/oae-core/editprofile/js/editprofile.js

```diff
@@ -23,9 +23,33 @@ define(['jquery', 'oae.core'], function ($, oae) {
         // Holds the current state of the user profile as it is updated
```

Comment:
As an aside, as soon as the v12 release is out, we need to ensure that accounts can only be created with the configured email domain for that tenant.